core/capabilities/gateway_connector/service_wrapper_test.go

```diff
@@ -1,4 +1,4 @@
-package gatewayconnector
+package gatewayconnector_test
```

Comment:
This is to avoid an import cycle caused by application.go and core/services/chainlink